lib/sanger_barcode_format/empty_barcode.rb

```diff
@@ -9,7 +11,7 @@ module SBCF
   #
   class EmptyBarcode
     # String representation
-    STRING_REP = '[empty]'.freeze
+    STRING_REP = '[empty]'
```

Comment:
why remove the `freeze` ?

Reply:
Because `# frozen_string_literal: true` is now at the top, so the string is frozen anyway